polynote-frontend/polynote/ui/component/notebook/comment.ts

```diff
@@ -431,7 +431,7 @@ class NewComment extends Disposable {
                 evt.preventDefault();
                 doCreate()
             }),
-            button(['cancel'], {}, ['Cancel']).mousedown(() => controls.classList.add("hide"))
+            button(['cancel'], {}, ['Cancel']).mousedown(() => {hide ? hide() : controls.classList.add("hide")})
```

Comment:
When the NewComment 'cancel' button is pressed, if a 'hide' function is passed in, it is called to remove the whole component. Otherwise, the controls (the 'cancel' and 'submit' buttons) will be hidden.

Reply:
Nice! If we wanted, we could maybe also clear the text in the comment text area specifically for replies (the 2nd branch of that ternary statement), so it both hides the controls and leaves a blank text box. Just an offhand thought, feel free to take or leave it!

Reply:
Great idea @mjren23! I just [added it](https://github.com/polynote/polynote/pull/1444/commits/12b2f502d2ccefd20360300d43e656fda1a4654d).